project/ParadoxSupport.scala

```diff
@@ -59,8 +56,18 @@ object ParadoxSupport {
       }
     }
 
+    private def baseClassName(label: String) = {
+      val labelWithoutGenerics = label.split("\\[")(0)
+      if (labelWithoutGenerics.endsWith("$")) labelWithoutGenerics.init
```

Comment:
Learned something here as I didn't know about `init` ;) Do you know where that terminology comes from?

Reply:
Not sure where it *originally* came from, I knew of it from Haskell... (init/last are kind of the "ugly cousins" of head/tail I suppose :) )

Reply:
Hehe, cool, thanks for the explanation :)